product.js

```diff
@@ -4,8 +4,7 @@ let b = 131;
 
 // function two find products of two numbers
 function findProduct(a, b) {
-  const prod = a * b;
-  return prod;
```

Comment:
This is the good style of coding this really works in reducing the space complexity.